lib/trilby.nix

```diff
@@ -146,6 +146,10 @@ rec {
         };
         imports = [
           inputs.self.nixosModules.home
+          inputs.nix-colors.homeManagerModule
+          ({ pkgs, ... }: {
+            colorScheme = inputs.nix-colors.colorSchemes.catppuccin-mocha;
+          })
```

Comment:
As you are not using them, you can omit the `{ pkgs, ... }:` here entirely; a module can have either of these types:
 - `AttrSet -> AttrSet`
 - `Attrset`